docs/dev-tools/chopsticks/overview.md

```diff
@@ -298,9 +289,7 @@ These are the methods that can be invoked and their parameters:
 
     === "Parameter"
 
-        |       Name       |               Type               | Description |
-        | :--------------: | :------------------------------: | :---------: |
-        | `buildBlockMode` | "Batch" \| "Instant" \| "Manual" | Build mode  |
+        - `buildBlockMode` (*"Batch", "Instant", or "Manual"*) - build mode
```

Comment:
So I think we should actually frame this a bit differently. The type it expects here is `BuildBlockMode`, which is an enum that exposes these options

One example of enum documentation in the XCM SDK docs: https://moonbeam-foundation.github.io/xcm-sdk/latest/reference/interfaces/#the-ecosystem-type

So maybe if we do something like this:

```
- `buildBlockMode` ++"BuildBlockMode"++ - the build mode. Can be any of the following modes as defined by the `BuildBlockMode` enum:
  \``` 
  export enum BuildBlockMode {
  Batch = 'Batch', /** One block per batch (default) */
  Instant = 'Instant', /** One block per transaction */
  Manual = 'Manual', /** Only build when triggered */
}
\```
```

wdyt?